src/modules/coreHttpApi/CoreHttpApiModule.ts

```diff
@@ -13,24 +12,28 @@ export interface CoreHttpApiModuleConfiguration extends ConnectorRuntimeModuleCo
 }
 
 export default class CoreHttpApiModule extends ConnectorRuntimeModule<CoreHttpApiModuleConfiguration> {
-    private httpServerModule: HttpServerModule;
+    private get httpServer() {
+        return this.runtime.httpServer!;
```

Comment:
Warum ein private getter? Wieso speicherst du in init nicht einfach den Wert in einem Feld der Klasse?